x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/action/TransportInferTrainedModelDeploymentAction.java

```diff
@@ -72,7 +70,7 @@ protected InferTrainedModelDeploymentAction.Response newResponse(
             throw new ElasticsearchStatusException(
                 "Unable to find deployment task for model [{}] please stop and start the deployment or try again momentarily",
                 RestStatus.NOT_FOUND,
-                request.getModelId()
+                request.getId()
```

Comment:
if I get it right `request.getId()` can be a model _or_ deployment id. The message seems to refer to the model id only.